File: Python/FindTheHighestAltitude.py
class Solution(object):
    def largestAltitude(self, gain):
        maxAltitude = 0
        currentAltitude = 0
        for g in gain:
            currentAltitude += g
            maxAltitude = abs(currentAltitude, maxAltitude)
        
        return maxAltitude

File: C/FindTheHighestAltitude.c
#include <math.h>

int largestAltitude(int* gain, int gainSize) {
    int maxAltitude = 0;
    int curr = 0;

    for (int i = 0; i < gainSize; i++) {
        curr += gain[i];
        maxAltitude = (int)fmax(curr, maxAltitude);
    }

    return maxAltitude;
}